docs/developing-workflow-tasks.md

```diff
@@ -0,0 +1,32 @@
+# Developing Workflow Tasks
+
+Workflow tasks can be either AWS Lambda Functions or ECS Activities.
+
+## Lambda functions
+
+You can use one of the many `@cumulus` node modules to take advantage of common functionality for Cumulus workflows, such as `@cumulus/parse-pdr` and `@cumulus/post-to-cmr`.
+
+A full set of available lambda functions can be found in [cumulus/tasks](https://github.com/cumulus-nasa/cumulus/tree/master/tasks).
```

Comment:
I think rather than link to github here, we should link to npm

Reply:
I think we should provide both links. The NPM link is not useful if you are looking for the source code.